docs/en/swip/SWIP-3.md

```diff
@@ -0,0 +1,60 @@
+
+### RocketMQ Topic Supported Metrics
+
+| Monitoring Panel                           |Unit        | Metric Name                                                             | Description                                                                                            | Data Source      |
+|--------------------------------------------|------------|-------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------|------------------|
+| Max Producer Message Size                  | Byte       | meter_rocketmq_topic_max_producer_message_size                          | The maximum number of messages produced.                                                               | RocketMQ Exporter |
+| Max Consumer Message Size                  | Byte       | meter_rocketmq_topic_max_consumer_message_size                          | The maximum number of messages consumed.                                                               | RocketMQ Exporter |
+| Consumer Latency                           | Ms         | meter_rocketmq_topic_consumer_latency                                   | Consumption delay time of a consumer group.                                                            | RocketMQ Exporter |
+| Producer Tps                               | Msg/sec    | meter_rocketmq_topic_producer_tps                                       | The number of messages produced per second.                                                            | RocketMQ Exporter |
+| Consumer Group Tps                         | Msg/sec    | meter_rocketmq_topic_consumer_group_tps                                 | The number of messages consumed per second per consumer group.                                         | RocketMQ Exporter |
+| Producer Offset                            | Count      | meter_rocketmq_topic_producer_offset                                    | The progress of a topic's production message.                                                          | RocketMQ Exporter |
+| Consumer Group Offset                      | Count      | meter_rocketmq_topic_consumer_group_offset                              | The progress of a topic's consumption message per consumer group.                                      | RocketMQ Exporter |
+| Producer Message Size                      | Byte       | meter_rocketmq_topic_producer_message_size                              | The size of messages produced per second.                                                              | RocketMQ Exporter |
+| Consumer Message Size                      | Byte       | meter_rocketmq_topic_consumer_message_size                              | The size of messages consumed per second.                                                              | RocketMQ Exporter |
```

Comment:
Where are the Max offset metrics?

Reply:
> Where are the Max offset metrics?

I originally thought that this metrics explanation did not need to be changed,it just change avg downsampling the max one ,ok, i will edit it

Reply:
The list here are logic meaning, so, Max indicates slowest or something in the description.